src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -37,12 +37,13 @@
 		<!-- Popover, counter and button(s) -->
 		<div v-if="item.utils" class="app-navigation-entry-utils">
 			<ul>
-				<!-- counter -->
-				<li v-if="Number.isInteger(item.utils.counter) && item.utils.counter > 0"
-					:class="{highlighted: item.utils.counter_highlighted}"
-					class="app-navigation-entry-utils-counter">
-					<span>{{ item.utils.counter }}</span>
-				</li>
+				<!-- counters -->
+				<AppNavigationCounter v-if="item.utils.mentionsCounter" :is-mention="true">
```

Comment:
Only one please.
```suggestion
				<AppNavigationCounter v-if="item.utils.mentionsCounter" :highligh="item.utils.counter_highlighted === true">
```

Reply:
> "Only one please."

Is this about the number of counters? At least two would really be prefered for Talk, so we can have a counter in blue for the mentions and one in grey for the unread messages.

There was additionally the idea to add a "camera"-icon in the same place when a call is happening. So not sure "one" is enough and suitable for that.

Reply:
> Is this about the number of counters? At least two would really be prefered for Talk, so we can have a counter in blue for the mentions and one in grey for the unread messages.

This is not needed and just a duplication of information.
Highlighting = mentionned. Number = unread message.
If you want to tell the user that there is a mention in the unread messages, it's best to use a highlighted counter. This is why we added this in the first place :)
Two counters are counter-productive ;)

Reply:
![cloud nextcloud com_call_g6ca5ka4](https://user-images.githubusercontent.com/14975046/61028993-582b7b80-a3ba-11e9-9ecb-e2a22c962d6d.png)